### DISCUSSION TITLE
A GUI for confidence scored words

### DISCUSSION THREAD

**Discussion by stevevaius2015** [linto-ai/whisper-timestamped#224] [Ideas]
Would be great to upload resulting JSON into a GUI to see each word's confidence by color (red to green) to easily check words and in case of wrong choice by model, easily click and change to correct one.

**Comment by misutoneko** [comment]
GUI-anything sounds like a separate project...

I suppose there are some simple cases where this could work,
like when the model gets the pronouns all confused (well who doesn't these days?)

How would you build the list of alternative words?
Collecting them during the inference is possible I guess, but there are problems with that approach.
